app/components/Stars/index.js

```diff
@@ -6,17 +6,39 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import ReactStars from 'react-stars';
 
+const EditDisabledOverlay = styled(ReactStars)`
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+`
+
 function Stars(props) {
   return (
-    <ReactStars
-      count={5}
-      value={props.value}
-      size={16}
-      edit={props.editable}
-      className={props.className}
-    />
+    <div className="position-relative">
+      <ReactStars
+        count={5}
+        value={props.value}
+        size={props.size}
+        edit={true}
+        className={props.className}
+        color1="lightgray"
+      >
+      </ReactStars>
+      {/* Had to create a controllable overlay, because ReactStars doesn't update its state
```

Comment:
If this component is tough to work with we can just use another stars library. This seems overly complicated. Thoughts?

Reply:
I have strongly considered that. Working through this has been an educational experience at least, though. I can look into another React stars library to use and would be glad to integrate it if you don't mind.

Reply:
Sure thing, let's wrap it like we did here. If this one is missing major functionality, go for it.